README.md

```diff
@@ -38,8 +38,8 @@ You can compress and decompress data.
 
 ```elixir
 iex> uncompressed = :crypto.strong_rand_bytes(10)
-iex> compressed = NimbleLZ4.compress(uncomppressed)
```

Comment:
causes `** (CompileError) iex:2: undefined function uncomppressed/0 (there is no such import)`